dbt/adapters/athena/impl.py

```diff
@@ -712,6 +712,7 @@ def list_relations_without_caching(self, schema_relation: AthenaRelation) -> Lis
                         LOGGER.debug(f"Table '{table['Name']}' has no TableType attribute - Ignoring")
                         continue
                     _type = table["TableType"]
+                    _detailed_table_type = table["Parameters"].get("table_type", None)
```

Comment:
Try to do `table.get("Parameters", {}).get("table_type")`

By default get return None. We could consider an empty string as well as default type.

Also, did you check that in Parameters we have table_type? I didn't find it in boto3 docs for get_tables

Reply:
`Parameters` - The user-supplied properties in key-value form ([boto3 docs get_tables](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue/client/get_tables.html#get-tables)). `table_type` may be present or not.

Reply:
Perfect, then if you adapt as suggested above the ci should work fine

Reply:
Unit tests were fixed, now it should works without fails.
Functional tests on the way. I guess, I will add them tomorrow.